news/2022-09-18-project-loved-september-2022.md

```diff
@@ -0,0 +1,349 @@
+---
+layout: post
+title: "Project Loved: September 2022"
+date: 2022-09-18 22:00:00 +0000
+---
+
+The captains at Project Loved present to you a carefully crafted curation of classic creations — or in other words, the next round is here! Come check out the maps and cast your votes.
+
+![](/wiki/shared/news/banners/project-loved-2.jpg)
+
+For those unfamiliar, Project Loved hosts monthly votings to decide which nominated maps get added to the [Loved category](https://osu.ppy.sh/beatmapsets?s=loved). You can click on each map's banner in this post to go to its poll on the forums, or visit [the forum](https://osu.ppy.sh/community/forums/120) for a list of all of the polls. Polls will be open for 10 days, so make sure to cast your votes in time! More info can be found on the [wiki page](/wiki/Community/Project_Loved) and [website](https://loved.sh)!
+
+Detailed results for the previous round can be found on the [poll results page of the website](https://loved.sh/statistics) or in the following forum posts:
+
+- [osu!](https://osu.ppy.sh/community/forums/posts/8768240)
+- [osu!taiko](https://osu.ppy.sh/community/forums/posts/8768239)
+- [osu!catch](https://osu.ppy.sh/community/forums/posts/8768238)
+- [osu!mania](https://osu.ppy.sh/community/forums/posts/8768237)
+
+The main updates this time around are about captain applications. First off, the osu! captains have closed their application form and decided on the next captains that'll be helping out. In the coming months, you'll get to know these fine gentlemen as they share maps here and open Loved polls:
+
+- [rHO](https://osu.ppy.sh/users/1629553)
+- [Mismagius](https://osu.ppy.sh/users/19048)
+- [Skill](https://osu.ppy.sh/users/7802027)
+- [DeviousPanda](https://osu.ppy.sh/users/4966334)
+
+Following the same idea, **the osu!catch captains have also opened an application**. Click this banner to apply if you're interested!
```

Comment:
it doesn't make sense here to say that *one* application has been "opened". applications are open

```suggestion
Following the same idea, **the osu!catch captains have also opened applications**. Click this banner to apply if you're interested!
```